monkvision/monkjs
# Feature/fe 268/drawer drag gestures

ALL CONTRIBUTIONS WILL BE REVIEWED BY AT LEAST ONE OF OUR FRONTEND TEAM MEMBERS.







## Technical description

Changed Drawer name to BottomSheet.
Added drag gestures to BottomSheet.

## Related to



This PR is related to [Update drawer(DamageCreateForm or ProfileEdit)](https://monkvision.atlassian.net/browse/FE-268)

## Context and Motivation 

Useful feature added is that the BottomSheet accepts a `lock` prop, which will lock it from being closed when it's `true`.

## About Tests


Tested on the following devices

- iOS physical device iPhone 8
- Android pixel 4 XL
- Chrome Web browser


Steps to reproduce

1. Add damage from damages.
2. Try swipe or close the BottomSheet from its dragging area (at the top).
3. Try closing the BottomSheet while the form is fulfilled.
4. Clear the form and try to close BottomSheet.
5. Try swipe the BottomSheet up (it should block you at some level).
6. The scrollview content should be fully rendered and not overflow.




## Screenshots (optional):

No screenshots.
*This Pull Request template has been written and generated by Monk JS repository.*
